pkgs/tools/security/ec2stepshell/default.nix

```diff
@@ -40,7 +43,7 @@ python3.pkgs.buildPythonApplication rec {
     description = "AWS post-exploitation tool";
     mainProgram = "ec2stepshell";
     homepage = "https://github.com/saw-your-packet/EC2StepShell";
-    changelog = "https://github.com/saw-your-packet/EC2StepShell/blob/${version}/CHANGELOG.txt";
+    changelog = "https://github.com/saw-your-packet/EC2StepShell/blob/${rev}/CHANGELOG.txt";
```

Comment:
if you put `src.rev` here instead of referring to the constant, you don't need to define the constant containing the git commit hash.